bus_board.rb

```diff
@@ -0,0 +1,71 @@
+require 'json'
+require 'net/http'
+require 'uri'
+
+def display_bus(bus)
+  time_until_arrival = bus["timeToStation"]/60 == 0 ? "due" : "#{(bus["timeToStation"]/60).to_s} minutes"
+  puts "line number: #{bus["lineName"]} || destination: #{bus["destinationName"]} || time until arrival: #{time_until_arrival}"
```

Comment:
Purely stylistic but maybe we could have a table like 
```
Line | Destination    | Time
134  | Kentish town |  due
134  | Kentish town |  5min
```
would need some work with padding/trimming so it's all even so up to you if you want it